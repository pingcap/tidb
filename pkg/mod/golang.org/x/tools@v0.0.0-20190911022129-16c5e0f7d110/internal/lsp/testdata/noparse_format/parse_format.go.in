package noparse_format //@format("package")

func _() {
f()
}