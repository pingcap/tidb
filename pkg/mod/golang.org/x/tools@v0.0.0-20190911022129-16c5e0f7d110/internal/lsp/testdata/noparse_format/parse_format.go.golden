-- gofmt --
package noparse_format //@format("package")

func _() {
	f()
}

-- gofmt-d --

